hugegraph-llm/src/hugegraph_llm/demo/rag_demo/vector_graph_block.py

```
@@ -43,114 +45,212 @@ def store_prompt(doc, schema, example_prompt):
+        selected_example = next((ex for ex in all_examples if ex.get("name") == example_name), None)
+
+        if selected_example:
+            return (
+                selected_example.get('description', ''),
+                selected_example.get('text', ''),
+                selected_example.get('prompt', ''),
+            )
+    except (FileNotFoundError, json.JSONDecodeError) as e:
+        log.warning("Could not update example preview: %s", e)
+    return "", "", ""
+def _create_prompt_helper_block(demo, input_text, info_extract_template):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)